internal/k8s/kubernetes.go

```diff
@@ -64,34 +48,6 @@ func DeletePodByLabel(podsClient coreV1Types.PodInterface, label string) {
 	}
 }
 
-// func CreateRepoSecret() {
-
-// }
-
-// func CreateCredentialsTemplateSecret() {
-
-// }
-
-func getVaultRootToken(vaultSecretClient coreV1Types.SecretInterface) string {
-	name := "vault-unseal-keys"
-	log.Printf("Reading secret %s\n", name)
-	secret, err := vaultSecretClient.Get(context.TODO(), name, metaV1.GetOptions{})
```

Comment:
Does this function really annoy us enough by existing? 

> maybe we can keep for a bit for reference code. 

> It is a nice example of marshal and secret manipulation.

Reply:
the function is not in use, there is no other part of the code that needs it. 

it will keep in git history in case we need it.

YAGNI is a good reference for that

Reply:
Approved with reservations, prefer we hold the deletion of artifacts. Details on approval.